Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
```

Comment:
That is part of it yes. This is intended specifically to begin tracing out those lines and figuring out how things should evolve. This initial take was meant just to give us some wording to kick around as we fill in the details.